scribble/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,58 @@
+class PostsController < ApplicationController
+  def index
+    @post = Post.all
```

Comment:
Convention here is to use `@posts` because we are referring to a collection of `posts`